NickvisionCavalier.Shared/Models/Configuration.cs

```diff
@@ -9,14 +10,10 @@ namespace NickvisionCavalier.Shared.Models;
 /// </summary>
 public class Configuration
 {
-    public static readonly string ConfigDir = $"{Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)}{Path.DirectorySeparatorChar}Nickvision{Path.DirectorySeparatorChar}{AppInfo.Current.Name}";
+    public static readonly string ConfigDir = $"{Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)}{Path.DirectorySeparatorChar}Nickvision{Path.DirectorySeparatorChar}{AppInfo.Current.ShortName}";
```

Comment:
When you meant getting rid of `Nickvision/Nickvision Application` path for example I made it like this:
```csharp
public static readonly string ConfigDir = $"{Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData)}{Path.DirectorySeparatorChar}{AppInfo.Current.Name}";
```
So the path will be just `CONFIG/Nickvision Application`. I think it's better than having it like you have it like this where it would be `CONFIG/Nickvision/Application`

Reply:
It's uncommon in FLOSS world, but I think it makes sense to use organization name as subfolder (like with `~/.mozilla`, `~/.local/share/JetBrains` and a bunch of proprietary stuff both on Windows and Linux).

Reply:
Ok, well the problem with `ShortName` is it's translatable....unlike `Name`....so if a translation changes the app name, the app will have different configs depending on the name for the language.

Reply:
Ah, true, will fix that